react/OrderFormUpdateHandler.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react'
+import type { OrderForm } from '@vtex/clients'
+
+import { notificationClient } from './clients/notificationClient'
+import { useOrderFormMonitor } from './hooks/useOrderFormMonitor'
+import { onEmailChanged } from './eventHandlers/onEmailChanged'
+import { useLoadScript } from './hooks/useLoadScript'
+import type { Fbq } from './typings/global'
+
+const OrderFormUpdateHandler: React.FC = () => {
+  const client = notificationClient()
+  const eventHandler = useOrderFormMonitor([onEmailChanged])
+  const handleOrderFormUpdated = async (_: unknown, orderForm: OrderForm) => {
+    const events = eventHandler(orderForm)
+
+    await client.notify(events)
+  }
+
+  const fbEventsLoaded = useLoadScript(
+    'https://connect.facebook.net/en_US/fbevents.js'
+  )
+
+  useEffect(() => {
+    if (fbEventsLoaded) {
+      $(window).on('orderFormUpdated.vtex', handleOrderFormUpdated)
+    }
+
+    return () => {
+      $(window).off('orderFormUpdated.vtex', handleOrderFormUpdated)
+    }
+  }, [eventHandler, fbEventsLoaded])
+
+  useEffect(() => {
+    window.fbq = function () {
```

Comment:
I think you can do this to avoid the lint warning:
```suggestion
    window.fbq = function fbq () {
```